.travis.yml

```diff
@@ -7,7 +7,7 @@ branches:
   - /^v[0-9]/
 env:
   global:
-  - KUBEVIRT_VER=v0.15.0-alpha.0
+  - KUBEVIRT_VER=v0.17.0
```

Comment:
PR is already created and waiting for merge https://github.com/kubevirt/common-templates/pull/65